### PR TITLE
complete animation button

### PR DESCRIPTION
**Pull request by chris-saburo**
#114 issue is fixed
- implemented animation effect on button
- rearrange nav bar buttons
- change numbers of auto-popup menu buttons

### CODE REVIEW

**Comment by paulchrisluke** [comment]
@chris-saburo the grey opacity layer should ease in

You can see an example of it here: 
gif
https://material.io/components/buttons-floating-action-button#types-of-transitions

File: src/components/BaseNavigationBottom.vue [hunk 1]
```diff
@@ -1,64 +1,65 @@
...
+                <BookingIcon class="m-auto" />
+                Article
               </router-link>
             </li>
           </ul>
           <img
             class="bottom-nav-icon"
             src="https://res.cloudinary.com/whynotearth/image/upload/v1588090427/Volkswagen/cms/Bottom_Nav_2x_vzwf6q.svg"
           />
+          <ul :class="{ ' opened': isOpen }" class="toggle-menu list-none">
```
Comment (atharva3010) [inline src/components/BaseNavigationBottom.vue:37]: why is there space before `opened`? Writing only `opened` works well.

File: src/components/BaseNavigationBottom.vue [hunk 2]
```diff
@@ -97,22 +102,64 @@ export default {
...
+
+.ease-in-out {
+  transition: all ease-in 0.2s;
+}
+
 .rotate-45 {
   transform: rotate(45deg);
 }
+
+ul.toggle-menu > li {
```
Comment (atharva3010) [inline src/components/BaseNavigationBottom.vue:141]: Please try to use tailwind utilities for most of the work. There is no need to write extra css.

File: src/components/BaseNavigationBottom.vue [hunk 3]
```diff
@@ -97,22 +102,64 @@ export default {
...
+  transition-duration: 450ms;
+  top: -70px;
+}
+
+ul.toggle-menu.opened > li:nth-child(2) {
+  transition-duration: 450ms;
+  top: -130px;
+}
+
+.arrange {
```
Comment (atharva3010) [inline src/components/BaseNavigationBottom.vue:158]: this all can be easily achieved by tailwind utilities

File: src/components/BaseNavigationBottom.vue [hunk 4]
```diff
@@ -97,22 +102,64 @@ export default {
...
+  display: flex;
+  justify-content: space-evenly;
+}
+
 .overlay {
   position: fixed;
```
Comment (atharva3010) [inline src/components/BaseNavigationBottom.vue:164]: Here too, most of the things can be done by tailwind utilities. I don't know who wrote this initially, but can you refactor this to use tailwind utilities?

File: src/components/BaseNavigationBottom.vue [hunk 5]
```diff
@@ -122,22 +169,27 @@ export default {
...
   opacity: 1 !important;
 }
+
 .router-link-exact-active svg path {
   fill: #03b3f9 !important;
   fill-opacity: 1 !important;
 }
+
 .router-link {
   color: white;
```
Comment (atharva3010) [inline src/components/BaseNavigationBottom.vue:184]: similarly for color

File: src/components/BaseNavigationBottom.vue [hunk 6]
```diff
@@ -122,22 +169,27 @@ export default {
...
 }
+
 .router-link-exact-active {
   color: #03b3f9 !important;
   opacity: 1 !important;
 }
+
 .router-link-exact-active svg path {
   fill: #03b3f9 !important;
   fill-opacity: 1 !important;
```
Comment (atharva3010) [inline src/components/BaseNavigationBottom.vue:180]: fill-opacity

File: src/components/BaseNavigationBottom.vue [hunk 7]
```diff
@@ -122,22 +169,27 @@ export default {
...
 }
+
 .router-link {
   color: white;
   opacity: 0.54;
 }
+
 .router-link svg path {
   fill: white;
```
Comment (atharva3010) [inline src/components/BaseNavigationBottom.vue:189]: fill: white

File: src/components/BaseNavigationBottom.vue [hunk 8]
```diff
@@ -97,22 +102,64 @@ export default {
 .bottom-nav-icon {
   height: 56px;
 }
+
 .toggle-button {
   width: 56px;
   height: 56px;
   border-radius: 50%;
   position: absolute;
```
Comment (atharva3010) [inline src/components/BaseNavigationBottom.vue:110]: absolute position and border-radius, it is all already included in tailwind utilities

File: src/components/BaseNavigationBottom.vue (hunk 4 above)
Comment (chris-saburo) [reply src/components/BaseNavigationBottom.vue]: I did refactor original css into tailwind css class. on possible cases.

File: src/components/BaseNavigationBottom.vue (hunk 1 above)
Comment (chris-saburo) [reply src/components/BaseNavigationBottom.vue]: I did fix it.

File: src/components/BaseNavigationBottom.vue (hunk 8 above)
Comment (chris-saburo) [reply src/components/BaseNavigationBottom.vue]: I did refactor about position and border radius into tailwind css class.

File: src/components/BaseNavigationBottom.vue (hunk 5 above)
Comment (chris-saburo) [reply src/components/BaseNavigationBottom.vue]: I did try to refactor it into tailwind css but it didn't work correctly.
So I did leave it there.

**Pull request merged**